tools/development/gen_mediapipe_libs.sh

```diff
@@ -4,20 +4,24 @@
 if [[ -z "${MEDIAPIPE_HOME}" ]]; then
   echo 'ERROR: Define MEDIAPIPE_HOME First!'
 else
-  # for the external objects
-  for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
-    ext_obj_arr+=($ext_obj)
-  done
-  $(gcc ${ext_obj_arr[@]} -shared -o libmediapipe_external.so -fPIC)
-  $(mv libmediapipe_external.so $1)
+  if [ -d "$1" ]; then
+    # `for the external objects
+    for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
+      ext_obj_arr+=($ext_obj)
+    done
+    $(gcc ${ext_obj_arr[@]} -shared -o libmediapipe_external.so -fPIC)
```

Comment:
Is it ok with the failure case of this line?

Reply:
the logic for the failure case has been updated